logfire/_internal/config_params.py

```diff
@@ -106,6 +106,8 @@ class _DefaultCallback:
 """Default sampling ratio for traces. Can be overridden by the `logfire.sample_rate` attribute of a span."""
 INSPECT_ARGUMENTS = ConfigParam(env_vars=['LOGFIRE_INSPECT_ARGUMENTS'], allow_file_config=True, default=sys.version_info[:2] >= (3, 11), tp=bool)
 """Whether to enable the f-string magic feature. On by default for Python 3.11 and above."""
+LOGFIRE_IGNORE_NO_CONFIG = ConfigParam(env_vars=['LOGFIRE_IGNORE_NO_CONFIG'], allow_file_config=True, default=False, tp=bool)
```

Comment:
```suggestion
IGNORE_NO_CONFIG = ConfigParam(env_vars=['LOGFIRE_IGNORE_NO_CONFIG'], allow_file_config=True, default=False, tp=bool)
```

The Python constant names here don't start with LOGFIRE_